resolwe_bio/tools/validate_master_file.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+"""Validate the uploaded master file."""
+
+import argparse
+import csv
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Run gene ontology enrichment analysis.")
+    parser.add_argument('master_file', type=str, help="Master file.")
+    return parser.parse_args()
+
+
+def check_dna_sequence(sequence):
+    """Check if a given sequence contains only the allowed letters A, C, T, G."""
+    for base in sequence:
+        if base in ['A', 'C', 'T', 'G']:
+            return True
```

Comment:
This is likely not what you intended to do, the `for` loop always stops at the first base.